CONTRIBUTING.rst

```diff
@@ -0,0 +1,345 @@
+
+.. contents:: Table of Contents
+..
+
+
+Introduction
+############
+
+You can contribute to Spinal Cord Toolbox in several ways:
+
+- Reporting issues you encounter
+
+- Providing new code
+
+- Contributing to the wiki or mailing list
+
+
+Reporting a bug or a requesting a feature
+#########################################
+
+
+Issues can be submitted `on our project's issue page
+<https://github.com/neuropoly/spinalcordtoolbox/issues>`_.
+
+
+Prior to submitting a new issue
+*******************************
+
+Consider the following:
+
+- Please take a few seconds to search the issue database in case the
+  issue has already been raised.
+
+- When reporting an issue, make sure your installation has not been tempered
+  with (and if you can, update to the latest release, maybe the problem was
+  fixed).
+
+
+When submitting an issue
+************************
+
+Consider the following:
+
+- Issue title: try to have a self-descriptive, meaningful issue title,
```

Comment:
(general comment): maybe something that would feel less heavy/boring in the reading of this document, would be to have **bold** format for key words, such as **Issue title**, *Body**, etc.

Reply:
I'll re-add subsections which should deal with that boldening. I don't know how I managed to have so many nested bullet points.